src/pages/Dashboard/Modals/BorrowRepay/Borrow/index.spec.tsx

```diff
@@ -41,6 +41,65 @@ describe('pages/Dashboard/BorrowRepayModal/Borrow', () => {
     renderComponent(<Borrow asset={fakeAsset} onClose={noop} isXvsEnabled />);
   });
 
+  it('renders correct token borrowable amount when asset liquidity is higher than maximum amount of tokens user can borrow before reaching their borrow limit', async () => {
+    const customFakeAsset: Asset = {
+      ...fakeAsset,
+      liquidity: new BigNumber(100000000),
+    };
+
+    const { getByText } = renderComponent(
+      <AuthContext.Provider
+        value={{
+          login: jest.fn(),
+          logOut: jest.fn(),
+          openAuthModal: jest.fn(),
+          closeAuthModal: jest.fn(),
+          account: {
+            address: fakeAccountAddress,
+          },
+        }}
+      >
+        <Borrow asset={customFakeAsset} onClose={noop} isXvsEnabled />
+      </AuthContext.Provider>,
+    );
+
+    const borrowDeltaDollars = fakeUserTotalBorrowLimitDollars.minus(
+      fakeUserTotalBorrowBalanceDollars,
+    );
+    const borrowDeltaTokens = borrowDeltaDollars.dividedBy(fakeAsset.tokenPrice);
+
+    await waitFor(() =>
+      getByText(`${borrowDeltaTokens.toFixed()} ${customFakeAsset.symbol.toUpperCase()}`),
```

Comment:
nit: `symbol` should always be the uppercase displayable value:

Reply:
Ok nice, I wasn't sure

Reply:
Fixed.